session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -558,6 +558,15 @@ public TSStatus createTimeseries(String path, TSDataType dataType,
     }
   }
 
+  public boolean checkTimeseriesExists(String path) throws IoTDBSessionException {
+    checkPathValidity(path);
+    try {
+      return executeQueryStatement(String.format("SHOW TIMESERIES %s", path)).hasNext();
```

Comment:
This is an interesting usage and it works...

Do we need to add an RPC interface for checking whether a time series exist?

Reply:
I think this is enough at present(as a tool method). If there are other inputting scenes later, we can file another PR to do that.